### PR TITLE
Create certificates for starting RESTAPI server

### PR DESCRIPTION
**Pull request by sumukhatv**
### Description of PR

RESTAPI server requires server certificates to be present in order to start. Without RESTAPI server starting, monit will complain that restapi is not running. Therefore, creating and deploying dummy certificates to start RESTAPI server, will prevent sanity checks from failing.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [*] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
RESTAPI server requires server certificates to be present in order to start. Without RESTAPI server starting, monit will complain that restapi is not running. Therefore, creating and deploying dummy certificates to start RESTAPI server, will prevent sanity checks from failing.
#### How did you do it?
In the config_sonic_basedon_testbed.yml, I added the openssl logic to create certificates on the DUT in the correct location.
#### How did you verify/test it?
New certificates were creates in /etc/sonic/credentials and restapi server started as expected.

### CODE REVIEW

**Review by prsunny — APPROVED** [review]

File: ansible/config_sonic_basedon_testbed.yml [hunk 1]
```diff
@@ -220,6 +220,52 @@
...
+      command: openssl req \
+          -x509 \
+          -sha256 \
+          -nodes \
+          -newkey rsa:2048 \
+          -keyout "{{ server_key }}"
+          -subj "/CN=sonic.restapi.test"
+          -out "{{ server_crt }}"
+      become: true
+
```
Comment (lguohan) [inline ansible/config_sonic_basedon_testbed.yml:268]: look at the telemetry certs deployment, it has 'deploy is defined and deploy|bool == true', why not have that for restapi certs deployment?

Comment (lguohan) [reply ansible/config_sonic_basedon_testbed.yml:268]: also, why is there difference between the telemetry cert and restapi cert?

Comment (lguohan) [reply ansible/config_sonic_basedon_testbed.yml:268]: if they are the essentially should be the same, then I think we need to unify them into one deployment yml.

Comment (sumukhatv) [reply ansible/config_sonic_basedon_testbed.yml:242]: I have moved the cert creation logic to one yml file.

**Review by lguohan — APPROVED** [review]

**Review by yxieca — APPROVED** [review]

**Pull request merged**